openrtx/include/protocols/M17/M17CodePuncturing.hpp

```diff
@@ -36,22 +37,73 @@ namespace M17
 /**
  * Puncture matrix for linx setup frame.
  */
-static constexpr auto LSF_PUNCTURE = std::experimental::make_array< uint8_t >
```

Comment:
Here the lint went crazy!

Reply:
Haha, going to have to use a `//clang-format off` here.